docs/server/source/data-models/transaction-model.rst

```diff
@@ -2,21 +2,3 @@ The Transaction Model
 =====================
 
 See the `IPDB Transaction Spec <https://github.com/ipdb/ipdb-tx-spec>`_.
```

Comment:
Should it refer to something under the BigchainDB organization?

Reply:
For now, the transaction spec / IPDB Transaction Spec lives under the `ipdb` organization. I think I'll eventually rename it and move it under the `bigchaindb` organization (e.g. maybe under the new /rfc repository), but I have higher-priority things to do first.